src/Chip/index.tsx

```diff
@@ -1,40 +1,61 @@
 import React, { ForwardedRef, KeyboardEventHandler, MouseEventHandler } from 'react';
-import PropTypes, { type Requireable } from 'prop-types';
 import classNames from 'classnames';
-// @ts-ignore
-import { requiredWhen } from '../utils/propTypes';
-import { STYLE_VARIANTS } from './constants';
 import ChipIcon from './ChipIcon';
 
 export const CHIP_PGN_CLASS = 'pgn__chip';
 
 export interface IChip {
+  /** Specifies the content of the `Chip`. */
   children: React.ReactNode,
+  /** Click handler for the whole Chip, has effect only when Chip does not have any interactive icons. */
```

Comment:
[optional]: By analogy with line `8` (Chip -> `Chip`)

```suggestion
  /** Click handler for the whole `Chip`, has effect only when Chip does not have any interactive icons. */
```